src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,52 @@
+.calendar {
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.